nanofi/include/core/cstructs.h

```diff
@@ -92,6 +102,17 @@ typedef struct {
  *
  */
 typedef struct {
+  // structural definitions for moving way from C++
+  unsigned modified:1;
```

Comment:
@arpadboda changes in cstructs were the only thing of merit here, where we have attributes...are you incorporating something like this into your site to site work?

Reply:
No, didn't touch C structures yet, I think I won't need to do either as there is C API to get attributes.